astroquery/mast/services.py

```diff
@@ -50,6 +50,7 @@ def _json_to_table(json_obj, data_key='data'):
 
     # for each item in info, store the type and column name
     # for each item in info, type has to be converted from DB data types (SQL server in most cases)
+    # from missions_mast search service such as varchar, integer, float, boolean etc
```

Comment:
could you be explicit here and mention that these keywords you are searching with are the `cols`?